2021q2/git.adoc

```diff
@@ -0,0 +1,49 @@
+=== Git Migration Working Group
+
+Links: +
+link:https://wiki.freebsd.org/Git[Git for FreeBSD development wiki] URL: link:https://wiki.freebsd.org/Git[https://wiki.freebsd.org/Git] +
+link:https://wiki.freebsd.org/GitTransition[Git transition wiki] URL: link:https://wiki.freebsd.org/GitTransition[https://wiki.freebsd.org/GitTransition] +
+link:https://cgit.FreeBSD.org/doc[doc git repo web] URL: link:https://cgit.FreeBSD.org/doc[https://cgit.FreeBSD.org/doc] +
+link:https://cgit.FreeBSD.org/ports[ports git repo web] URL: https://cgit.FreeBSD.org/ports[https://cgit.FreeBSD.org/ports] +
+link:https://cgit.FreeBSD.org/src[src (base system) git repo web] URL: link:https://cgit.FreeBSD.org/src[https://cgit.FreeBSD.org/src] +
+link:https://docs.freebsd.org/en/articles/committers-guide/#git-primer[Committers guide Git primer] URL: link:https://docs.freebsd.org/en/articles/committers-guide/#git-primer[https://docs.freebsd.org/en/articles/committers-guide/#git-primer] +
+link:https://docs.freebsd.org/en/books/handbook/mirrors/#git[Handbook Using Git appendix] URL: link:https://docs.freebsd.org/en/books/handbook/mirrors/#git[https://docs.freebsd.org/en/books/handbook/mirrors/#git] +
+link:http://gameoftrees.org/[Game of Trees] URL: link:http://gameoftrees.org/[http://gameoftrees.org/] +
+link:https://github.com/johnmehr/gitup[gitup] URL: link:https://github.com/johnmehr/gitup[https://github.com/johnmehr/gitup]
+
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Warner Losh <imp@FreeBSD.org> +
+Contact: Ed Maste <emaste@FreeBSD.org> +
+Contact: Ulrich Spörlein <uqs@FreeBSD.org> +
+Contact: link:https://lists.freebsd.org/mailman/listinfo/freebsd-git[FreeBSD-git mailing list] +
+Contact IRC #gitcvt channel on EFnet
+
+The Git Working Group has been working on ports repository migration to Git, this task started at the end of the March, beginning with a final Subversion commit on March 31st to indicate that the conversion started.
+The whole migration completed on April 6th.
+Since 2021Q2, the ports quarterly branch is created in Git repository only.
+We continued working on portsnap and other ports infrastructure to accommodate git.
+
+We continued working on implementing and updating commit hooks.
+The FreeBSD 13.0 release process was using Git
```

Comment:
Needs either a full stop at the end or the "And" on next line to be changed to "and". Also, I think s/was using/used/